include/zephyr/bluetooth/byteorder.h

```diff
@@ -107,6 +107,78 @@ extern "C" {
 	BT_BYTES_LIST_LE32(_v),        \
 	BT_BYTES_LIST_LE32((_v) >> 32) \
 
```

Comment:
The compliance check suggestion has a valid point here, i.e. right now the empty line after the define is actually still part of it due to the `\` on what should be the last line

Reply:
Should I apply the suggestion?

Reply:
Ys, please :)

Reply:
That applies to all of the above macros, right?

Why is compliance even checking this line when it's not changed @pdgendt?

Reply:
@alwa-nordic @Thalley oh, sorry - I completely overlooked that part, i.e. that the line wasn't changed in this PR. Please ignore my comment (in the context of this PR).

Reply:
Right, the clang format performs checks on the diff, so it might take the 3 lines before and after into account, will look into this.

Reply:
Weirdly enough, I don't get the same behavior locally:

```shell
$ ./scripts/ci/check_compliance.py --annotate -m ClangFormat
Running ClangFormat      tests in /home/pdgendt/zephyrproject/zephyr ...

Complete results in compliance.xml
```

Reply:
> Right, the clang format performs checks on the diff, so it might take the 3 lines before and after into account, will look into this.

I think how many lines `git` checks before and after a changed line can be locally configured (defaults to 3). Maybe there is somehow a difference there between your local setup and what is running this?

Reply:
But the script already passes `-U0` to `git diff` so it has no before or after changes.

Reply:
Strange. I guess we'll just have an keep an eye on it manually then :)